android/libraries/rib-android/src/test/kotlin/com/uber/rib/core/RibActivityTest.kt

```diff
@@ -189,6 +190,17 @@ class RibActivityTest {
     assertThat(activity.interactor).isNotNull()
   }
 
+  @Test
+  fun onCreate_setsViewTreeOwners() {
+    val activity = Robolectric.buildActivity(EmptyActivity::class.java).create(null).get()
+    assertThat(activity.window.decorView.findViewTreeLifecycleOwner()).isNotNull()
+
+    val contentView = activity.findViewById<FrameLayout>(android.R.id.content)
+    val rootView = View(RuntimeEnvironment.application)
+    contentView.addView(rootView)
+    assertThat(rootView.findViewTreeLifecycleOwner()).isNotNull()
```

Comment:
Can we assert for `ViewTreeSavedStateRegistry` and `ViewTreeViewModelStoreOwner` as well?

Reply:
Added more assertions in 72b5415